HospitalSystem.ConsoleUI/Doctor/DoctorMainUI.cs

```diff
@@ -149,32 +149,26 @@ public void PrintOperations(List<Operation> operations)
 
     public void RequestDaysOff()
     {
-        Console.Write("\nEnter desired range for off days\nStarting date >>");
-        string? start = Console.ReadLine();
-        Console.Write("\nEnding date >>");
-        string? end = Console.ReadLine();
-        var startDate = DateTime.TryParse(start, out DateTime newStartDate);
-        var endDate = DateTime.TryParse(end, out DateTime newEndDate);
-        if (startDate == true && endDate== true && newStartDate > DateTime.Now)
+        DateRange daysOff = InputDateRange();
+        if (daysOff.Starts > DateTime.Now)
         {
-           CreateRequest(newStartDate,newEndDate);
+           CreateRequest(daysOff);
         }
         else
             Console.WriteLine("Invalid date, cannot be before today.");
```

Comment:
I think input DateRange already checks for this? Maybe?

Reply:
Oh, it does, didn't realise. Thanks!